define_all_flags.c

```diff
@@ -141,6 +141,18 @@ struct flags_parser *add_flags_tcp_stream(struct flags_parser *fp)
         DEFINE_FLAG(fp, bool,          enable_write,    false,   'w', "Write to flows? Enabled by default for the client");
         DEFINE_FLAG(fp, bool,          enable_tcp_maerts,    false,   'M', "Enables TCP_MAERTS test (server writes and client reads). It overrides enable_read, and enable_write");
         DEFINE_FLAG(fp, bool,          async_connect,   false,   0,  "use non blocking connect");
+#ifdef WITH_TCPDEVMEM
+        DEFINE_FLAG(fp, bool,          tcpd_validate,   false,  0, "Validates that received data is a repeating sequence of 1 to 111 inclusive");
```

Comment:
Not critical at all. But if possible, consider adding a check to make sure that rx_cpy flag is set if validation is enabled. Either report an error or automatically overrides the `tcpd_rx_copy` flag